test/integration/submission/submission.spec.ts

```diff
@@ -35,14 +35,17 @@ import { TsvUtils } from '../../../src/utils';
 import { donorDao } from '../../../src/clinical/donor-repo';
 import { Donor } from '../../../src/clinical/clinical-entities';
 import AdmZip from 'adm-zip';
-
-import * as _ from 'lodash';
+import _ from 'lodash';
 import { SchemasDictionary } from '../../../src/lectern-client/schema-entities';
 import { DictionaryMigration } from '../../../src/submission/schema/migration-entities';
-
+import * as util from 'util';
 chai.use(require('chai-http'));
 chai.should();
 
+const baseDonorId = 250000;
+const baseSampleId = 610000;
```

Comment:
I think this is mixed with baseSpecimenId

Reply:
correct, thanks